tests/golden/check-ignored/check-ignored.bats

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bats
+
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+
+@test "Ignore file with broken xrefcheck annotation: full path" {
+  run xrefcheck --ignored ./to-ignore/broken_annotation.md
+
+  assert_output --partial "All repository links are valid."
+}
+
+@test "Ignore file with broken xrefcheck annotation: glob wildcard" {
+  run xrefcheck --ignored 'to-ignore/*'
+
+  assert_output --partial "All repository links are valid."
+}
+
+@test "Ignore file with broken xrefcheck annotation: config file" {
+  run xrefcheck --config ./config-ignored.yaml
+
+  assert_output --partial "All repository links are valid."
+}
```

Comment:
Great tests! :+1: can you add some for `notScanned` as well please?

Reply:
Oh, while adding these tests, I noticed that `notScanned` doesn't actually filter directories... Thanks for helping me notice that!  :smile: